proposals/lip-0066.md

```diff
@@ -45,6 +45,16 @@ We define the following constants:
 | `PrivateKeyEd25519` | bytes  | Byte sequences of length `ED25519_PRIVATE_KEY_LENGTH`. | An Ed25519 private key. |
 | `PrivateKeyBLS`     | bytes  | Byte sequences of length `BLS_PRIVATE_KEY_LENGTH`. | An BLS private key. |
 
+## bytesToInt Conversion
+For the `chainID` , we use the bytes value. We define the function `bytesToInt` to deserialize bytes to integer value.
+
+```python
+def bytesToInt(numberBytes: bytes) -> int:
```

Comment:
we want to convert chainID to an integer, why isnt the input just `chainID`?

Reply:
Updated the function. Although, input could be anything.